### PR TITLE
Fixes #11638 - Removing deprecated code for Katello 3.0 release

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

**Comment by ehelms** [comment]
This change looks good -- there are a number of other depredations that we could remove given the bump to 3.0. Thoughts on tackling them as one PR or separate PRs?

**Comment by daviddavis** [comment]
Updated with all of the outstanding 3.0 deprecation work. Some notes:

- Activation key subscription routes are still being used by UI
- Not touching systems controller. Most of it is still being used and I don't feel qualified to remove these routes/params.

File: app/controllers/katello/api/v2/content_view_filters_controller.rb [hunk 1]
```diff
@@ -70,10 +70,6 @@ def destroy
 
     private
 
-    def deprecated
```
Comment (daviddavis) [inline app/controllers/katello/api/v2/content_view_filters_controller.rb]: AFAICT, this is not being used anymore.

**Comment by daviddavis** [comment]
@ehelms ping.

**Comment by ehelms** [comment]
ACK

**Pull request merged**